modular_skyrat/modules/conveyor_sorter/code/conveyor_sorter.dm

```diff
@@ -1,12 +1,13 @@
 /obj/item/conveyor_sorter
 	name = "conveyor sorter lister"
-	desc = "A tool that is used to not only create the conveyor sorters, but give lists to the conveyor sorters."
+	desc = "A tool that is used to not only create the conveyor sorters, but give lists to the conveyor sorters. It has a limit of four sorters"
 	icon = 'modular_skyrat/modules/conveyor_sorter/icons/conveyor_sorter.dmi'
 	icon_state = "lister"
 	///the list of conveyor sorters spawned by
 	var/list/spawned_sorters = list()
 	///the list of things that are currently within the sorting list
 	var/list/current_sort = list()
+	var/max_sorters = 4
```

Comment:
Document this variable please.

It's done by adding `/// A description of what the variable represents/is used for`